.circleci/config.yml

```diff
@@ -1,21 +1,24 @@
 version: 2.1
 
 parameters:
+  resource_class:
+    type: string
+    default: "macos.x86.medium.gen2"
```

Comment:
The new resource class from the [announcement](https://discuss.circleci.com/t/macos-resource-deprecation-update/46891), defined as a parameter to get injected into each job.